long-live-pomf.md

```diff
@@ -26,16 +26,16 @@
 | https://sugoi.vidyagam.es/ | Up               |                                               |              100 | Nice colors, pastebin
 | https://vidga.me/          | Up               |                                               |              100 | JS-free
 | https://yiff.moe/          | Down             |                                               |              512 | Nice colors, metadata stripping
-| https://pomf.sinister.ly/  | Up               |                                               |              100 |
+| http://pomf.sinister.ly/   | Up               |                                               |              100 | Not secure - http only
```

Comment:
`Not secure` is redundant, as it applies to all http sites. Keep in mind that http entries are supposed to be below all https ones.